custom/abdm/milestone_one/views/abha_creation_views.py

```diff
@@ -0,0 +1,54 @@
+from rest_framework.decorators import api_view, permission_classes, authentication_classes
+from rest_framework.permissions import IsAuthenticated
+from custom.abdm.auth import UserAuthentication
+
+from custom.abdm.milestone_one.utils import abha_creation_util as abdm_util
+from custom.abdm.milestone_one.utils.decorators import required_request_params
+from custom.abdm.milestone_one.utils.response_handler import get_response
+
+
+@api_view(["POST"])
+@permission_classes((IsAuthenticated,))
+@authentication_classes((UserAuthentication,))
+@required_request_params(["aadhaar"])
+def generate_aadhaar_otp(request):
+    aadhaar_number = request.data.get("aadhaar")
+    resp = abdm_util.generate_aadhar_otp(aadhaar_number)
```

Comment:
should this be named "resp" or this is payload?

Reply:
This is raw response from abdm call. We process the received response further, in next line.

Reply:
raw_response looks like a better name so its clear what would be returned.

Reply:
bumping this comment

Reply:
https://github.com/dimagi/commcare-hq/pull/32332/commits/001fb505d52feaf56b3ee170703a4d8a64c1346d